skpro/distributions/alpha.py

```diff
@@ -0,0 +1,67 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Alpha probability distribution."""
+
+__author__ = ["SaiReavanth25"]
+
+import pandas as pd
+from scipy.stats import alpha, rv_continuous
+
+from skpro.distributions.adapters.scipy import _ScipyAdapter
+
+
+class Alpha(_ScipyAdapter):
+    r"""Alpha distribution.
+
+    The alpha distribution is characterized by its shape parameter :math:`\a`,
```

Comment:
could you add one of the distribution defining functions here, as a reference? pdf for instance

Reply:
![image](https://github.com/sktime/skpro/assets/132150745/7b540b2a-26ed-4291-9a4e-9471da1a1d2f)

Does this check out?

![Screenshot 2024-05-24 132836](https://github.com/sktime/skpro/assets/132150745/af2caadf-1655-44a3-97b7-e4461c15c910)

Reply:
Almost:

* I think the inline tex expression might have to be on the same line to compile. Have a look how it render in the doc build.
* I would not make the dependency on `a` explicit on the lfet hand side, for consistency
* the `\` in front of the `a` is unnecessary
* `x` is not a random variable, it's a free (bound) variable. You can omit its explanation.
* you can also omit the argument of `Phi` in the explanation, and say "is" instead of "represents".

Reply:
made the necessary changes, i think its good to go.